lib/proofOfPatience.js

```diff
@@ -130,7 +131,7 @@ api.verifyToken = function(ip, did, token, callback) {
         }
 
         // decode and verify the token
-        token = jwt.decode(token, key.data);
+        token = jwt.verify(token, key.data, {clockTolerance: 250});
```

Comment:
We should make `clockTolerance` a configurable variable ... and probably set it to like 5 seconds (I realize this means any PoP wait time value less than this renders the proof mechanism useless). It's probably more reasonable to be at like 30 seconds, but that starts to eat into our ability to have a responsive service. We should also add an issued time claim (`iat`) so that the client can compare that time against their own clock and make adjustments (for smarter clients).